src/components/ControlTable.vue

```diff
@@ -62,138 +138,117 @@
+    var arr = temp.contextualControls.map((item: any) => {
+      return item.data;
+    });
+    for (var i = 0; i < arr.length; i++) arr[i] = hdfWrapControl(arr[i]);
+    return arr;
+  }
+  getColor(def: string) {
+    var color;
+    switch (def) {
+      case "Passed":
+        color = "#0f0";
+        break;
+      case "Failed":
+        color = "#f00";
+        break;
+      case "Not Applicable":
+        color = "#00f";
+        break;
+      case "Not Reviewed":
+        color = "#888";
+        break;
+      case "Profile Error":
+        color = "#000";
+        break;
+      default:
+        color = null;
+    }
+    return color;
+  }
+  getNist(nist: any) {
+    return nist.join(", ");
   }
   expanded: any[] = [];
   search: string = "";
   singleExpand: boolean = false;
+  var: boolean = false;
   headers: Header[] = [
     {
-      text: "Dessert (100g serving)",
+      text: "Status",
       align: "left",
-      sortable: false,
-      value: "name"
-    },
-    { text: "Calories", value: "calories" },
-    { text: "Fat (g)", value: "fat" },
-    { text: "Carbs (g)", value: "carbs" },
-    { text: "Protein (g)", value: "protein" },
-    { text: "Iron (%)", value: "iron" }
-  ];
-  desserts: Item[] = [
-    {
-      name: "Frozen Yogurt",
-      calories: 159,
-      fat: 6.0,
-      carbs: 24,
-      protein: 4.0,
-      iron: "1%"
-    },
-    {
-      name: "Ice cream sandwich",
-      calories: 237,
-      fat: 9.0,
-      carbs: 37,
-      protein: 4.3,
-      iron: "1%"
-    },
-    {
-      name: "Eclair",
-      calories: 262,
-      fat: 16.0,
-      carbs: 23,
-      protein: 6.0,
-      iron: "7%"
-    },
-    {
-      name: "Cupcake",
-      calories: 305,
-      fat: 3.7,
-      carbs: 67,
-      protein: 4.3,
-      iron: "8%"
-    },
-    {
-      name: "Gingerbread",
-      calories: 356,
-      fat: 16.0,
-      carbs: 49,
-      protein: 3.9,
-      iron: "16%"
-    },
-    {
-      name: "Jelly bean",
-      calories: 375,
-      fat: 0.0,
-      carbs: 94,
-      protein: 0.0,
-      iron: "0%"
+      value: "status"
     },
-    {
-      name: "Lollipop",
-      calories: 392,
-      fat: 0.2,
-      carbs: 98,
-      protein: 0,
-      iron: "2%"
-    },
-    {
-      name: "Honeycomb",
-      calories: 408,
-      fat: 3.2,
-      carbs: 87,
-      protein: 6.5,
-      iron: "45%"
-    },
-    {
-      name: "Donut",
-      calories: 452,
-      fat: 25.0,
-      carbs: 51,
-      protein: 4.9,
-      iron: "22%"
-    },
-    {
-      name: "KitKat",
-      calories: 518,
-      fat: 26.0,
-      carbs: 65,
-      protein: 7,
-      iron: "6%"
-    }
+    { text: "Title", value: "wraps.title" },
+    { text: "Control ID", value: "vuln_num" },
+    { text: "Severity", value: "severity" },
+    { text: "NIST SP 800-53", value: "nist_tags" }
   ];
 }
 </script>
+
+<style scoped>
+.newline {
+  white-space: pre;
+}
+.wset {
+  min-width: 125px;
+  justify-content: center;
+}
+
+.test {
```

Comment:
Applying this class to the prism code highlighting sections leads to the code highlighted section not fitting its wrapping container, and since there is a border on the wrapping container, this behavior is very obvious

See the following image where the test class is removed from the top two code blocks

<img width="1165" alt="Screen Shot 2019-08-16 at 10 28 46 AM" src="https://user-images.githubusercontent.com/40170735/63175256-a6631880-c011-11e9-841d-6fa06fe74db4.png">